### PR TITLE
feature: add tweet-retrieve-api-with-comments

### PR DESCRIPTION
**Pull request by W-Shih**
- feature: add tweet-retrieve-api-with-comments
- feature: update Comment ordering
- test: add tests for tweet-retrieve-api-with-comments

### CODE REVIEW

File: tweets/api/views.py [hunk 1]
```diff
@@ -31,14 +36,22 @@ class TweetViewSet(viewsets.GenericViewSet):
     # rest_framework will use serializer_class to be the POST format at the rest page
     # - https://www.django-rest-framework.org/api-guide/generic-views/#genericapiview
     serializer_class = TweetSerializerForCreate
+    queryset = Tweet.objects.all()
```
Comment (wen-wayne-shih) [inline tweets/api/views.py:39]: TODO: prefetch_related for user

File: tweets/api/serializers.py [hunk 2]
```diff
@@ -33,6 +35,17 @@ class Meta:
         fields = ('id', 'user', 'created_at', 'content')
 
 
+class TweetSerializerWithComments(TweetSerializer):
+    # <Wayne Shih> 26-Nov-2021
+    # - https://docs.djangoproject.com/en/3.2/topics/db/queries/#following-relationships-backward
+    # - https://www.django-rest-framework.org/api-guide/fields/#using-source
+    comments = CommentSerializer(source='comment_set', many=True)
```
Comment (wen-wayne-shih) [inline tweets/api/serializers.py:42]: TODO: prefetch_related for comments

**Review by wen-wayne-shih — COMMENTED** [review]
TODO: prefetch_related

File: tweets/api/views.py (hunk 1 above)
Comment (W-Shih) [reply tweets/api/views.py:39]: Will do it in the next follow-up PR.

File: tweets/api/serializers.py (hunk 2 above)
Comment (W-Shih) [reply tweets/api/serializers.py:42]: Will do it in the next follow-up PR.

**Review by wen-wayne-shih — APPROVED** [review]

**Pull request merged**